test/pages/login/login_controller_test.dart

```diff
@@ -46,6 +46,7 @@ void main() {
 
       expect(onFailedCalled, true);
       expect(loginController.isLoading.value, false);
+      loginController.onClose();
```

Comment:
@sleepylee oh, is this recommended from GetX? 🙏 I think this `onClose()` function should be in `tearDown()` to cover all test cases in a group 😄

Reply:
actually, it is supposed to be called once the instance is removed from GetX dependencies graph 🥲 I'm not sure why doesn't it triggered in my case even when I tried to call `Get.delete<LoginController>()` 😢  
probably will move to tearDown yeah

Reply:
@manh-t check it out, I figured out the lifecycle of the GetxController, so I invoked put/delete to indirectly verify the `onClose()` method